fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -4604,14 +4604,19 @@ show_param ::=
         RESULT = new ShowSyncJobStmt(dbName);
     :}
     /* show table stats */
-    | KW_TABLE opt_cached:cached KW_STATS table_name:tbl opt_partition_names:partitionNames opt_col_list:cols
```

Comment:
why remove show cached?

Reply:
Because it's useless, table stats is all stored in memory. The action of `show table cached stats` is identical with `show table stats`. 
Besides, there will be parsing conflict if we don't remove this opt_cached